### ISSUE TITLE
Standalone Time Stamp Margin does not work in VS2017 RTM

### ISSUE THREAD

**Issue by ktos** [Microsoft/VS-PPT#97]
Just installed VS 2017 Community RTM and now standalone Time Stamp Margin, and there is unfortunately no effect. 

All options are available in Tools -> Options ->Productivity Power Tools, but nothing is show in Output window.

I tried turning the option off and on again, but it not fixed this ;-)

**Comment by gigios** [comment]
I have the same issue!

**Comment by olegtk** [comment]
Hmmm, it works ok for me. Note that this extension only adds timestamp to the Debug Output Window pane.

**Comment by gigios** [comment]
Ok! Probably I have found the problem!
In my case I have also the extension VSColorOutput to 'decorate' the debug output lines. If I disable this extension I'm able to show the Time Stamp Margin, if I re-enable the extension the margin disappears.

The VSColorOutput has a similar functionality but I prefer the Time Stamp Margin. Probably when this function is disabled i the extension, also other extensions are not able to add the margin on the output window.
I will try to send the same question to the developer of the VSColorOutput.
For the moment thanks for the support.

**Comment by AdamDotNet** [comment]
gigios is right, Time Stamp Margin does not work when VSColorOutput is installed. I really like having both, I hope something can be worked out.

**Comment by justcla** [comment]
Until a fix is delivered, I recommend using the TimeStamp margin feature built into VSColorOutput as a workaround.

"As of VSColorOutput 2.2.1 you can optionally show timestamps in your output."

![image](https://cloud.githubusercontent.com/assets/17131343/26486538/5dbef9f0-41b0-11e7-9c69-457a7baa81ab.png)

**Comment by AdamDotNet** [comment]
@justcla That's true, and I have turned it on. I guess I'm just used to Power Tools' format/behavior more. The formats used by VSColorOutput and Power Tools each have their pros and cons. :)

**Comment by metablaster** [comment]
I want to add that even after disabling VSColorOutput the stand alone timestamp margin didn't work for me.

VS 16.5.4
